src/modules/components/header.js

```diff
@@ -0,0 +1,10 @@
+export default function header() {
+  const header = document.createElement('section');
+  header.className = 'title padding-x';
+  header.innerHTML = `
+  <div class="header">
+    <h1 class="display-1">Leaderboard</h1>
+  </div>`;
+
+  return header;
+}
```

Comment:
- [ ] Let's remind ourselves that we are working with `webpack` and `ES6` features in this project. Please Let's refactor this function to use ES6 syntax.